### PR TITLE
add onboarding

### PR DESCRIPTION
**Pull request by mroz22**

### CODE REVIEW

File: packages/suite-onboarding/static/fonts/fonts.css [hunk 1]
```diff
@@ -0,0 +1,287 @@
+@font-face {
```
Comment (vladimirvolek) [inline packages/suite-onboarding/static/fonts/fonts.css:1]: Load font from suite core

File: packages/suite/src/components/onboarding/GlobalWebNavigation/components/LanguagePicker/index.tsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import React from 'react';
```
Comment (vladimirvolek) [inline packages/suite/src/components/onboarding/GlobalWebNavigation/components/LanguagePicker/index.tsx]: language picker is already in suite

File: packages/suite/src/components/onboarding/GlobalWebNavigation/index.messages.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import { defineMessages } from 'react-intl';
```
Comment (vladimirvolek) [inline packages/suite/src/components/onboarding/GlobalWebNavigation/index.messages.tsx]: Move global navigation to suite

File: packages/suite/src/config/onboarding/colors.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
+// todo: some of them are not used
```
Comment (vladimirvolek) [inline packages/suite/src/config/onboarding/colors.ts:1]: use colors from components

File: packages/suite/src/config/onboarding/languages.ts [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import { Languages } from '@suite/types/onboarding/languages';
```
Comment (vladimirvolek) [inline packages/suite/src/config/onboarding/languages.ts]: use languages from suite

File: packages/suite/src/config/onboarding/urls.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
+export const SUPPORT_URL = 'https://trezor.io/support/';
```
Comment (vladimirvolek) [inline packages/suite/src/config/onboarding/urls.ts:1]: move constants to suite

File: packages/suite/src/types/onboarding/languages.ts [hunk 7]
```diff
@@ -0,0 +1,26 @@
+type Code =
```
Comment (vladimirvolek) [inline packages/suite/src/types/onboarding/languages.ts]: move to suite core

File: packages/suite/src/components/onboarding/GlobalWebNavigation/components/LanguagePicker/index.tsx (hunk 2 above)
Comment (mroz22) [reply packages/suite/src/components/onboarding/GlobalWebNavigation/components/LanguagePicker/index.tsx]: Not applicable now, as we have temporary build target "suite-onboarding"

File: packages/suite/src/components/onboarding/GlobalWebNavigation/index.messages.tsx (hunk 3 above)
Comment (mroz22) [reply packages/suite/src/components/onboarding/GlobalWebNavigation/index.messages.tsx]: Yes, should be removed.

File: packages/suite/src/components/onboarding/GlobalWebNavigation/components/LanguagePicker/index.tsx (hunk 2 above)
Comment (mroz22) [reply packages/suite/src/components/onboarding/GlobalWebNavigation/components/LanguagePicker/index.tsx]: done

File: packages/suite/src/config/onboarding/languages.ts (hunk 5 above)
Comment (mroz22) [reply packages/suite/src/config/onboarding/languages.ts]: removed.

File: packages/suite/src/config/onboarding/urls.ts (hunk 6 above)
Comment (mroz22) [reply packages/suite/src/config/onboarding/urls.ts:1]: done with PR adding links to header.

**Pull request merged**